dev-scheme/racket/racket-8.1.ebuild

```diff
@@ -0,0 +1,184 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop optfeature pax-utils xdg-utils
+
+DESCRIPTION="General purpose, multi-paradigm Lisp-Scheme programming language"
+HOMEPAGE="https://racket-lang.org/"
+SRC_URI="
+	minimal? ( https://download.racket-lang.org/installers/${PV}/${PN}-minimal-${PV}-src-builtpkgs.tgz )
+	!minimal? ( https://download.racket-lang.org/installers/${PV}/${P}-src-builtpkgs.tgz )
+"
+S="${WORKDIR}/${P}/src"
+
+# See https://blog.racket-lang.org/2019/11/completing-racket-s-relicensing-effort.html
+LICENSE="
+	|| ( MIT Apache-2.0 )
+	chez? ( Apache-2.0 )
+	!chez? ( LGPL-3 )
+"
+# Bytecode generated by Racket is not compatible between versions
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="X +chez +doc +futures +jit minimal +places +readline +threads"
+REQUIRED_USE="futures? ( jit )"
+
+DEPEND="
+	!dev-tex/slatex
+	dev-db/sqlite:3
+	dev-libs/libffi:=
+	X? (
+		dev-util/desktop-file-utils
+		media-libs/libpng:0
+		virtual/jpeg:0
+		x11-libs/cairo[X]
+		x11-libs/gtk+:3[X]
+		x11-libs/pango[X]
+		x11-misc/shared-mime-info
+	)
+	readline? ( dev-libs/libedit )
+"
+RDEPEND="${DEPEND}"
+
+# "mred" and "mzscheme" are binaries generated by Racket, not CC
+QA_FLAGS_IGNORED="usr/bin/mred usr/bin/mzscheme"
+
+# Backports that will be probably available in 8.3 release
+PATCHES=(
+	# Prevent "libracket3m" from being stripped
+	# https://github.com/racket/racket/commit/92d202060201f569fafc348c4f64a2965b55287f
+	"${FILESDIR}/Makefile-nostrip.patch"
+	# Fix "starter" LDFLAGS
+	# https://github.com/racket/racket/commit/4f0e1703979d704c4f91b009945c324450acec7a
+	"${FILESDIR}/Makefile-starter.patch"
+)
+
+post_X_update() {
+	if use X && ! use minimal; then
+		xdg_desktop_database_update
+		xdg_icon_cache_update
+	fi
+}
+
+src_prepare() {
+	# Prepare environment
+	unset PLTADDONDIR PLTCOLLECTS PLTCONFIGDIR PLTUSERHOME
+	xdg_environment_reset
+
+	default
+
+	# Remove bundled libffi
+	rm -r ./bc/foreign/libffi || die "failed to remove bundled libffi"
+}
+
+src_configure() {
+	# According to vapier, we should use the bundled libtool
+	#   such that we don't preclude cross-compile.
+	#   Thus don't use --enable-lt=/usr/bin/libtool
+	# --enable-bc builds Racket w/o chez backend
+	local myconf=(
```

Comment:
No need for an intermediate variable here. These options can be passed to `econf` directly.

Reply:
sure they can, but it looks awful i my hones opinion with larger configurations, where we need to escape newline `\`